src/Type.php

```diff
@@ -642,4 +642,14 @@ public function schema()
         $this->schema = new MappingSchema($name, $type->getMapping());
         return $this->schema;
     }
+
+    /**
+     * The default connection name to inject when creating an instance.
+     *
+     * @return string
+     */
+    public static function defaultConnectionName()
+    {
+        return 'default';
```

Comment:
I wonder if it should be `elastic` or something different to `default`, that would make it easier for us to integrate with the rest of cakephp